pandas/core/arrays/_mixins.py

```diff
@@ -176,9 +182,14 @@ def _concat_same_type(
         return to_concat[0]._from_backing_data(new_values)  # type: ignore[arg-type]
 
     @doc(ExtensionArray.searchsorted)
-    def searchsorted(self, value, side="left", sorter=None):
-        value = self._validate_searchsorted_value(value)
-        return self._ndarray.searchsorted(value, side=side, sorter=sorter)
+    def searchsorted(
+        self,
+        value: ArrayLike | object,
+        side: Literal["left", "right"] = "left",
+        sorter: NumpySorter = None,
+    ) -> np.ndarray:
+        npvalue: np.ndarray = cast(np.ndarray, self._validate_searchsorted_value(value))
```

Comment:
i dont think validate_searchsorted_value wraps scalars i ndarrays does it?

Reply:
No, it doesn't.  Fixed in next commit by typing `validate_searchsorted_value`